pkg/kubelet/container/runtime.go

```diff
@@ -364,14 +364,23 @@ type PortMapping struct {
 	HostIP string
 }
 
+type DeviceInfo struct {
+	// Path on host for mapping
+	PathOnHost string
+	// Path in Container to map
+	PathInContainer string
+	// Cgroup permissions
+	Permissions string
```

Comment:
@vishh I'm aware of that we have mount struct for the volume, and I think maybe we can have another struct for device mapped to container, like the GPU devices, because this is also an important information in container, or we only need the number of GPU mapped to container? I'm working on be familiar with the new API now.